test/integration/targets/pull/pull-integration-test/local.yml

```diff
@@ -1,5 +1,6 @@
 - name: test playbook for ansible-pull
   hosts: all
+  gather_facts: False
```

Comment:
Should we also have a `gather_facts: True` test case?

Reply:
that is default

Reply:
Right, but there's only one test play here and it's `gather_facts: False`, I was suggesting that there be a second play that does gather facts or am I missing something?

Reply:
i was just going for 'faster play' as if tasks normally succeed, there is no reason to think that fact gathering would fail, it executes under ansible-playbook in any case and we already have other tests for fact gathering.

Reply:
👍